qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -18,6 +27,11 @@
             nic_model_nic1 = virtio
             nic_model_nic2 = rtl8139
             device_name = "Red Hat VirtIO Ethernet Adapter"
+            variants:
+                - @default:
+                - with_floopy:
+                    run_with_floopy = yes
+                    vol_virtio_key = "DeviceId like 'A:'"
```

Comment:
vol_virtio_key parameter can be moved to the top of the .cfg file to reduce the repeated definition

Reply:
lijin, have a default vol_virtio_key param at the top of the cfg, it used to test single driver install from iso, here vol_virtio_key will rewrite the default value when test single driver install from floopy.

Reply:
Oh,I see.
Thanks for explanation.